app-backup/amanda/amanda-3.5.1-r3.ebuild

```diff
@@ -188,145 +186,170 @@ src_configure() {
+	myconf+=( $(use_with kerberos krb5-security) )
 
 	# Amazon S3 support
-	myconf="${myconf} `use_enable s3 s3-device`"
+	myconf+=( $(use_enable s3 s3-device) )
 
 	# libcurl is required for S3 but otherwise optional
 	if ! use s3; then
-		myconf="${myconf} $(use_with curl libcurl)"
+		myconf+=( $(use_with curl libcurl) )
 	fi
 
 	# Client only, as requested in bug #127725
-	if use minimal ; then
-		myconf="${myconf} --without-server"
+	if use minimal; then
+		myconf+=( --without-server )
 	else
 		# amplot
-		myconf="${myconf} $(use_with gnuplot)"
+		myconf+=( $(use_with gnuplot) )
 	fi
 
 	# IPv6 fun.
-	myconf="${myconf} `use_with ipv6`"
+	myconf+=( $(use_with ipv6) )
 	# This is to prevent the IPv6-is-working test
 	# As the test fails on binpkg build hosts with no IPv6.
 	use ipv6 && export amanda_cv_working_ipv6=yes
 
 	# I18N
-	myconf="${myconf} `use_enable nls`"
+	myconf+=( $(use_enable nls) )
 
 	# Bug #296634: Perl location
 	perl_set_version
-	myconf="${myconf} --with-amperldir=${VENDOR_LIB}"
+	myconf+=( --with-amperldir="${VENDOR_LIB}" )
 
 	# Bug 296633: --disable-syntax-checks
 	# Some tests are not safe for production systems
-	myconf="${myconf} --disable-syntax-checks"
+	myconf+=( --disable-syntax-checks )
 
 	# build manpages
-	myconf="${myconf} --enable-manpage-build"
+	myconf+=( --enable-manpage-build )
 
 	# Bug #636262
-	myconf="${myconf} $(use_with ndmp ndmp)"
+	myconf+=( $(use_with ndmp ndmp) )
 
 	# bug #483120
 	tc-export AR
 
 	econf \
+		--disable-static \
 		$(use_with readline) \
-		${myconf}
+		"${myconf[@]}"
 }
 
 src_compile() {
 	# Again, do not want the perl-module src_compile
-	default_src_compile
+	default
+}
+
+# We do not want the perl-module tests.
+src_test() {
+	default
+}
+
+# We have had reports of amanda file permissions getting screwed up.
+# Losing setuid, becoming too lax etc.
+# ONLY root and users in the amanda group should be able to run these binaries!
+amanda_permissions_fix() {
+	local root="$1"
+	[[ -z ${root} ]] && die "Failed to pass root argument to amanda_permissions_fix!"
+	local i le=/usr/libexec/amanda
+	for i in "${le}"/calcsize "${le}"/killpgrp "${le}"/rundump "${le}"/runtar; do
+		[[ -e ${root}/${i} ]] || continue
+		chown root:${AMANDA_GROUP_NAME} "${root}"/${i} || die
+		chmod u=srwx,g=rx,o= "${root}"/${i} || die
+	done
+
+	# amanda-security.conf is a config file with similar requirements:
+	# writable only by root
+	# world-readable
+	# 3.3.9: introduced in /etc/amanda-security.conf
+	# 3.4.2: moved to /etc/amanda/amanda-security.conf
+	f=/etc/amanda/amanda-security.conf
+	chown root:root "${root}""${f}" || die
+	chmod u=rw,go=r "${root}""${f}" || die
 }
 
 src_install() {
-	[ ! -f "${TMPENVFILE}" ] && die "Variable setting file (${TMPENVFILE}) should exist!"
+	[[ ! -f ${TMPENVFILE} ]] && die "Variable setting file (${TMPENVFILE}) should exist!"
 	source ${TMPENVFILE}
 
 	einfo "Doing stock install"
 	emake DESTDIR="${D}" install
```

Comment:
"${ED}" if we are everywhere else?

Reply:
Nope, `${EPREFIX}` has already been added in `econf`